### DISCUSSION TITLE
Albers Equal Area Projection?

### DISCUSSION THREAD

**Discussion by diehl** [opengeos/leafmap#977] [Q&A] [answered]
Is it possible to use other projections besides the default of EPSG:4326? I'm interested in using an Albers equal area projection for CONUS if possible.

**Comment by giswqs** [comment, marked as answer]
See https://github.com/jupyter-widgets/ipyleaflet/blob/master/examples/CustomProjections.ipynb

**Reply by diehl** [reply]
@giswqs Does this imply custom projections are not an option when using MapLibre?

**Reply by giswqs** [reply]
Correct. MapLibre does not support custom projections yet. See https://maplibre.org/roadmap/non-mercator-projection/

**Reply by diehl** [reply]
Glad to see its on the roadmap - thanks @giswqs ! Much appreciated.
